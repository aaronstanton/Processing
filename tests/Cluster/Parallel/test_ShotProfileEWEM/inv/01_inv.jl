using Seismic

param = {"Niter"=>5,"cost"=>"cost.txt","mu"=>[0.0 0.0 0.0],"precon"=>false,
	"vp"=>"../prep/vp", "vs"=>"../prep/vs", "wav"=>"../prep/wav",
	"angx"=>"../prep/angx","angy"=>"../prep/angy",
	"sz"=>10, "gz"=>550,
	"fmin"=>1., "fmax"=>50.,
	"padt"=>2, "padx"=>2,
	"nangx"=>21, "oangx"=>-50, "dangx"=>5,
	"nangy"=>1, "oangy"=>0, "dangy"=>5,
	"nhx"=>827, "ohx"=>-3304, "dhx"=>8,
	"nhy"=>1, "ohy"=>0, "dhy"=>8,
	"verbose"=>true}
param["sx"] = [100.0:100.0:6900.0];
param["sy"] = [100.0:100.0:6900.0]*0;
m0 = ["../adj/mpp";"../adj/mps1";"../adj/mps2"]
m = ["mpp";"mps1";"mps2"]
d = ["../prep/ux";"../prep/uy";"../prep/uz"]
ShotProfileLSEWEM(m,m0,d,param);

