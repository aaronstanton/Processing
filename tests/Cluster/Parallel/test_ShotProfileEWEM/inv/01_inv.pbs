#!/bin/bash 
#PBS -S /bin/bash
#PBS -l nodes=3:ppn=12 
#PBS -l mem=66gb 
#PBS -l walltime=24:00:00 
#PBS -m bea 
#PBS -M aaron.stanton@ualberta.ca
#PBS -N 01_inv
#PBS -o 01_inv.out
#PBS -e 01_inv.err

date

# Change to directory from which job was submitted
cd $PBS_O_WORKDIR 

echo "here is your PBS_NODEFILE"
cat $PBS_NODEFILE

echo "calling julia now"
julia -p 3 --machinefile $PBS_NODEFILE 01_inv.jl

