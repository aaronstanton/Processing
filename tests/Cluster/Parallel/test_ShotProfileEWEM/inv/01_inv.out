Sat Nov 21 14:16:15 MST 2015
here is your PBS_NODEFILE
cn0733
cn0733
cn0733
cn0733
cn0733
cn0733
cn0733
cn0733
cn0733
cn0733
cn0733
cn0733
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0740
cn0740
cn0740
cn0740
cn0740
cn0740
cn0740
cn0740
cn0740
cn0740
cn0740
cn0740
calling julia now
